lib/Models/WebMapServiceCatalogItem.js

```diff
@@ -332,6 +333,21 @@ var WebMapServiceCatalogItem = function(terria) {
     "_allLayersInRawMetadata"
   ]);
 
+  knockout.defineProperty(this, "_colorRange", {
+    get: function() {
+      return this.colorScaleMaximum + "  " + this.colorScaleMinimum;
+    }
+  });
+
+  knockout.getObservable(this, "_colorRange").subscribe(function(_colorRange) {
```

Comment:
This isn't ideal because every WMS catalog ever, even ones that are just in the catalog, will subscribe to this property. But I guess fixing this is more trouble than it's worth. In the new model architecture we won't have to subscribe at all, it'll just work.